internal/util/paramtable/component_param.go

```diff
@@ -1004,6 +1006,22 @@ func (p *queryCoordConfig) init(base *BaseTable) {
 		PanicIfEmpty: true,
 	}
 	p.UpdateNextTargetInterval.Init(base.mgr)
+
+	p.CheckNodeInReplicaInterval = ParamItem{
+		Key:          "queryCoord.checkNodeInReplicaInterval",
+		Version:      "2.2.3",
+		DefaultValue: "60",
+		PanicIfEmpty: true,
+	}
+	p.CheckNodeInReplicaInterval.Init(base.mgr)
+
+	p.EnableQueryNodePoolMode = ParamItem{
+		Key:          "queryCoord.enableQueryNodePoolMode",
+		Version:      "2.2.3",
+		DefaultValue: "false",
+		PanicIfEmpty: true,
```

Comment:
Shall not panicIfEmpty. Will cause user instance crash loop if upgrade from lower version